content/hi/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,167 @@
+---
+title: नियंत्रकों
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+रोबोटिक्स और ऑटोमेशन में, एक _कंट्रोल लूप_ होता है
+एक नॉन-टर्मिनेटिंग लूप जो एक सिस्टम की स्थिति को नियंत्रित करता है।
+
+यहां कंट्रोल लूप का एक उदाहरण दिया गया है: कमरे में थर्मोस्टेट।
+
+जब आप तापमान सेट करते हैं, तो वह थर्मोस्टेट को बताता है
+आपकी *वांछित अवस्था* के बारे में। वास्तविक कमरे का तापमान है
+*वर्तमान स्थिति*। थर्मोस्टेट वर्तमान स्थिति लाने के लिए कार्य करता है
+उपकरण को चालू या बंद करके वांछित स्थिति के करीब।
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+<!-- body -->
+
+## नियंत्रक पैटर्न
```

Comment:
Consider incorporating English text for the link to avoid non-English characters in the URL of the section.

```suggestion
## नियंत्रक पैटर्न {#controller-pattern}
```